test/rounding.jl

```diff
@@ -134,6 +134,15 @@ end
         end
     end
 
+    for Tf in [Float16,Float32,Float64]
+        @test round(eps(Tf)/8, RoundNearest) === zero(Tf)
+        @test round(-eps(Tf)/8, RoundNearest) === -zero(Tf)
+        @test round(eps(Tf)/8, RoundNearestTiesAway) === zero(Tf)
+        @test round(-eps(Tf)/8, RoundNearestTiesAway) === -zero(Tf)
+        @test round(eps(Tf)/8, RoundNearestTiesUp) === zero(Tf)
+        @test round(-eps(Tf)/8, RoundNearestTiesUp) === -zero(Tf)
+    end
```

Comment:
What about testing each rounding behavior at the high and low value of each possible value of the exponent? That should be reasonably quick and fairly thorough. I know that it's probably overkill but, why not?